test/components/dataset/settings.spec.js

```diff
@@ -0,0 +1,102 @@
+import DatasetPendingSubmissions from '../../../src/components/dataset/pending-submissions.vue';
+
+import testData from '../../data';
+import { load } from '../../util/http';
+import { mockLogin } from '../../util/session';
+
+describe('DatasetSettings', () => {
+  beforeEach(mockLogin);
+
+  it('should have onReceipt selected', async () => {
+    testData.extendedProjects.createPast(1);
+    testData.extendedDatasets.createPast(1);
+    const component = await load('/projects/1/datasets/trees/settings');
+    component.get('input[value="true"]').element.checked.should.be.false();
+    component.get('input[value="false"]').element.checked.should.be.true();
+  });
+
+  it('should have onApproval selected', async () => {
+    testData.extendedProjects.createPast(1);
+    testData.extendedDatasets.createPast(1, { approvalRequired: true });
+    const component = await load('/projects/1/datasets/trees/settings');
+    component.get('input[value="true"]').element.checked.should.be.true();
+    component.get('input[value="false"]').element.checked.should.be.false();
+  });
+
+  it('should sends the correct request', async () => {
+    testData.extendedProjects.createPast(1);
+    testData.extendedDatasets.createPast(1);
+    await load('/projects/1/datasets/trees/settings', { root: false })
+      .complete()
+      .request(async (component) => component.get('input[value="true"]').trigger('change'))
```

Comment:
I think `setValue(true)` would be better here than `trigger('change')`. `setValue(true)` will trigger a `change` event, but it will also set the `checked` property of the element. As far as I know, `trigger()` just triggers the event.